hacking.md

```diff
@@ -0,0 +1,303 @@
+# Developing SPO
+This document describes how to build, install and test SPO for development
+purposes. It is not exhaustive - knowledge of how an operator works is
+presumed and PRs are always welcome.
+
+## Building SPO locally
+Even though SPO is a Kubernetes operator and as such is normally not meant to
+be used locally, but rather deployed in a cluster from pre-built images,
+it's often useful to be able to run a quick build or a test on your local machine.
+
+Depending on what OS and version you are developing on and what features you
+want to build with, you might need to either install extra dependencies or
+disable features that would require them.
+
+There are currently two optional features at build time
+- eBPF based recording
+  - This feature requires a rather new `libelf` and `libbpf` versions
```

Comment:
To be fully complete, libbpf depends on libelf which depends on libz

Reply:
ok, fixed (hopefully :-))